pkg/env/costmodelenv.go

```diff
@@ -239,6 +240,12 @@ func GetInsecureSkipVerify() bool {
 	return GetBool(InsecureSkipVerify, false)
 }
 
+// IsCacheDisabled returns the environment variable value for DisableCache which
+// will inform the aggregator on whether to load cached data. Defaults to false
+func IsCacheDisabled() bool {
```

Comment:
@Pokom would you be open to renaming this function and the associated variable `IsAggregateCostModelCacheDisabled()` and `DISABLE_AGGREGATE_COST_MODEL_CACHE`, respectively?

Reply:
Certainly! I just pushed a commit to update the references to be explicit in what cache is being set. In addition to the method definitions, I updated the other `disableCache` variables to be `disableAggregateCostModelCache`. Let me know if that's a problem!

Reply:
Not at all -- very helpful, thank you.